html/editing/editing-0/contenteditable/contentEditable-shortcuts.html

```diff
@@ -0,0 +1,53 @@
+<!DOCTYPE html>
+<html>
+<head>
+    <title>Editing: manual test for keyboard shortcuts with contenteditable=plaintext-only</title>
```

Comment:
This comment says "manual test" but the file name doesn't have "-manual" suffix, and this looks like an automated test. Can you verify which this is?